src/assets/styles/base.scss

```diff
@@ -276,3 +276,12 @@ ul.list {
 .vue-affix {
   width: 282px;
 }
+
+// Overrides the chip style for 'Corrected' label used across components
+.v-size--x-small {
```

Comment:
Very cool, I did update the V-chip slightly in my recent PR but it shouldn't have any conflict here :)